File: 21567.c
#include <stdio.h>
int main(){
    int a,b,c;
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);
    char res[20];
    sprintf(res,"%lld",(long long int)a*b*c);
    int count[10]={0};
    for (int i=0; res[i]!=0; i++)count[res[i]-48]++; 
    for (int i=0; i<10; i++) printf("%d\n", count[i]);
    return 0;
}

File: 21567.py
#파이썬으로 시간초과나면 C로
a=int(input())
b=int(input())
c=int(input())
res=list(str(int(a*b*c)))
count=[0 for _ in range(10)]
for i in res: count[int(i)]+=1
for i in count: print(i)